reactor-core/src/test/java/reactor/ReactorTestExecutionListener.java

```diff
@@ -61,11 +63,19 @@ private static void resetHooksAndSchedulers() {
 	@Override
 	public void executionFinished(TestIdentifier testIdentifier, TestExecutionResult testExecutionResult) {
 		resetHooksAndSchedulers();
+		if (testIdentifier.getTags().contains(TestTag.create("changesLoggerFactory"))) {
```

Comment:
Seems a bit blunt to use a tag and custom logic in this class. Why not `@After` or `@AfterAll` in `LoggersTest` only?

Reply:
centralize the logic of setting up the logger factory in the `TestExecutionListener` only

Reply:
Could also just call `LoggerUtils.useCurrentLoggersWithCapture();` once more at the end of `LoggerUtilsTest.disposeOnlyUninstallsItelf()`, seems way more simple and understandable. wdyt?

Reply:
is one really better than the other? note that right now if we were to do the reset in an `@AfterAll`, it would first have to reset the logger to default one (since the current logger could be eg. jdkLogger), then reinstall capturing logger on top of it.

Reply:
pushed a commit to see the diff. if approved I'll rework the PR's history and do the merge.

Reply:
I went and did the rebase in order to get 4 meaningful commits